From 16ed7e39a116a41e8c71895d8f2c604636299cd0 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Mon, 10 Sep 2018 18:27:43 +0800
Subject: [PATCH 4579/5242] MLK-19518-2: HDP API: continues running set mode
 function if ddc failed

commit  20925c42a5cd17b723e9a6e45c509f8dd1991e60 from
https://source.codeaurora.org/external/imx/linux-imx.git

Continues running set mode function to support
edid read failed case.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/hdp/API_HDMITX.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/mxc/hdp/API_HDMITX.c b/drivers/mxc/hdp/API_HDMITX.c
index da8d511..f9706d8 100644
--- a/drivers/mxc/hdp/API_HDMITX.c
+++ b/drivers/mxc/hdp/API_HDMITX.c
@@ -171,9 +171,8 @@ CDN_API_STATUS CDN_API_HDMITX_READ_EDID_blocking(state_struct *state, u8 block,
 		pr_info("Skip DDC Write for iMX8QM A0 SOC\n");
 	else {
 		ret = CDN_API_HDMITX_DDC_WRITE_blocking(state, &data_in, &data_out);
-		pr_info("CDN_API_HDMITX_DDC_WRITE_blocking ret = %d\n", ret);
 		if (ret != CDN_OK)
-			return ret;
+			pr_warn("CDN_API_HDMITX_DDC_WRITE_blocking ret = %d\n", ret);
 	}
 
 	ret = CDN_API_General_Read_Register_blocking(
-- 
1.7.9.5

